IO/BeagleBone/BBBPin.cs

```diff
@@ -78,7 +78,7 @@ static class Pin
             { BBBPin.P9_21, new PinData(0x954, 85, 3, GpioEnum.GPIO_3, 0) },
             { BBBPin.P9_22, new PinData(0x950, 84, 2, GpioEnum.GPIO_2, 0) },
             { BBBPin.P9_23, new PinData(0x844, 17, 49, GpioEnum.GPIO_49, 0) },
-            { BBBPin.P9_24, new PinData(0x984, 97, 15, GpioEnum.GPIO_15, 0) },
+            { BBBPin.P9_24, new PinData(0x984, 97, 15, GpioEnum.GPIO_15, 0) }, // 
```

Comment:
What are the slashes at the end of the line? (Just curious)

Reply:
Honestly not sure what this one was for, but further down it's to emphasize that there is a gap in pin numbers.